### ISSUE TITLE
Kneeboard position in NON VR mode with multiple monitor

### ISSUE THREAD

**Issue by fdallera** [OpenKneeboard/OpenKneeboard#258]
Hi,
I have 4 monitors in DCS,  monitor 1 2 3 are inline and are the DCS 3D viewport monitor 4 is for the Helios instruments and is under the monitors 2 thats it means that the Screen area for DCS is 5760 x  2160

i would like to have the Kneeboard on the near the left edge of the right monitor (number 2) but there is no way ...
if i set "minimum distance from window edge, in pixels"  the kneebord is moved diagonally  in this way the kneebord disappered on the bottom
 
![immagine](https://user-images.githubusercontent.com/18698792/216078647-9655ad55-608c-4b6f-a523-431f0acbedd4.png)

**Comment by fredemmott** [comment]
I'm currently leaning towards keeping the current settings, but adding an optional way to specify the bounding box - for example, if all of your monitors are 1024x768 and you want it on:

- monitor 3: specify (1024, 0) to (2048, 768)
- monitor 4: specify (1024, 768) to (2048, 1536)

... with the top left corner (of monitor 2, in your case) being (0, 0)

**Comment by kbastronomics** [comment]
I have the same issue, recently asked on the discord and was mentioned to change the offset but that it's only a diagonal move is the issue.   if we could move X/Y separately we'd have allot more positional control and you won't have to worry about monitors i would think

**Comment by OloRatuj** [comment]
Indeed, it makes OpenKneebord unusable in the multimonitor setup. +1 to the free X/Y move option

**Comment by fredemmott** [comment]
A workaround is to use [the viewer](https://openkneeboard.com/how-to/OpenKneeboard-Viewer/) in combination with the [always on top powertoy](https://learn.microsoft.com/en-us/windows/powertoys/always-on-top), instead of the current non-VR support.

Use the powertoy to make it always on top, 'f' to switch the background fill to fully transparent, 'b' for borderless mode.
